pandas/io/excel/_base.py

```diff
@@ -400,7 +400,7 @@ def parse(self,
             data = self.get_sheet_data(sheet, convert_float)
             usecols = _maybe_convert_usecols(usecols)
 
-            if sheet.nrows == 0:
+            if not data:
```

Comment:
What was the requirement for this change?

Reply:
sheet.nrows is an abstraction leakage of xlrd, an openpyxl sheet does not have this attribute. I think checking for if there is data is more explicit end general.